src/SerialInputHandler.cpp

```diff
@@ -125,3 +144,25 @@ void SerialInputHandler::Update() {
     }
   }
 }
+
+void SerialInputHandler::PrintWelcomeHeader() {
+  Serial.println("\n============== OPENSHOCK ==============");
+  Serial.println("  Contribute @ github.com/OpenShock");
+  Serial.println("  Discuss @ discord.gg/AHcCbXbEcF");
+  Serial.println("  Type 'help' for available commands");
+  Serial.println("=======================================\n");
+}
+
+void SerialInputHandler::PrintVersionInfo() {
+  Serial.print("  Version:  ");
+  Serial.println(OPENSHOCK_FW_VERSION);
+  Serial.print("    Build:  ");
+  Serial.println(OPENSHOCK_FW_MODE);
+  Serial.print("   Commit:  ");
+  Serial.println(OPENSHOCK_FW_COMMIT);
+  Serial.print("    Board:  ");
+  Serial.println(OPENSHOCK_FW_BOARD);
+  Serial.print("     Chip:  ");
+  Serial.println(OPENSHOCK_FW_CHIP);
```

Comment:
This could a bunch of printf statements instead:
Serial.printf("   Example: %s\n", OPENSHOCK_EXAMPLE);

Would this make it more readable in your opinion? Used to doing it like this

Reply:
You could also just do this since they are string literals, but that might be confusing:
`Serial.println(" Example: " OPENSHOCK_EXAMPLE);`